### PR TITLE
Add icons for capitals of admin_level 5 and 6

### LINKED ISSUE #384: Distinguish county seats from other cities
Currently, state capitals are marked by a star, but many of the same maps that denote state capitals this way also have a special icon for county seats. We should also use a distinct icon, especially at zoom levels where county boundaries aren’t visible yet. County seats are tagged `capital=6`.

Benchmark Arizona

### PR DESCRIPTION
**Pull request by claysmalley**
Fixes #384

This PR refactors the capital symbology to apply to all values of `place=*` (not just `city`), and assigns places with `capital=5` and `capital=6` a new icon, `place_dot_in_circle`.

[image](https://preview.ourmap.us/pr/1078/#map=8.07/32.752/-97.064)

[image](https://preview.ourmap.us/pr/1078/#map=7.04/33.438/-118.158)

[image](https://preview.ourmap.us/pr/1078/#map=6.25/28.501/76.786)

### CODE REVIEW

File: src/layer/place.js [hunk 1]
```diff
@@ -451,4 +457,14 @@ export const legendEntries = [
...
+  {
+    description: "Division capital",
+    layers: populatedPlaceLayers,
+    filter: ["==", ["get", "capital"], 5],
+  },
+  {
+    description: "County seat or local capital",
+    layers: populatedPlaceLayers,
+    filter: ["==", ["get", "capital"], 6],
+  },
```
Comment (claysmalley) [inline src/layer/place.js:469]: Legend labels for `capital=*` are gonna depend heavily on the country. Not sure what the best approach is here.

Comment (jleedev) [reply src/layer/place.js:469]: New York City is a "Division" capital 🤔

**Comment by ZeLonewolf** [comment]
Overall PR looks good, but I'm seeing some minor artifacts on my display and wondering if we need to mess with the scaling a bit?  I note that this icon is at a different scale than the state and national capital icons in the sprite sheet.  See map example:

image

**Comment by claysmalley** [comment]
I made the whitespace slightly thicker in the icon. Does this look better on your screen?

image

**Comment by ZeLonewolf** [comment]
Definite improvement here:
image

**Review by ZeLonewolf — APPROVED** [review]
Looks great!

**Pull request merged**